docs/src/tools.rst

```diff
@@ -63,6 +63,10 @@ Delete all database content with :github-source:`tools/prune_database.sh`::
 
     ./tools/prune_database.sh
 
+| Update testdata file based on the *current* database with :github-source:`tools/update_test_data.sh`  
+**Disclaimer**: make sure to first run ``prune_database.sh`` and make *only* the desired changes before running this tool::  
+
+    ./tools/update_test_data.sh
 
```

Comment:
```suggestion
Update testdata file based on the *current* database with :github-source:`tools/update_test_data.sh`  
**Disclaimer**: make sure to first run ``prune_database.sh`` and make *only* the desired changes before running this tool::  

    ./tools/update_test_data.sh

.. _translations:


```

Reply:
Thank you for the suggestion, but apparently the lineblock is necessary to have a line break before the disclaimer. 
The linter only required a newline **after** the lineblock.
I have fixed this, and the checks are passing now.